content/blog/vlan-versus-vxlan.md

```diff
@@ -0,0 +1,81 @@
+---
+title: VLAN Versus VXLAN
+date: 2024-11-11T14:57:37.063Z
+author: Indira Manthri
+authorimage: /img/indira_manthri.jpg
+disable: false
+tags:
+  - aruba
+  - aruba-central
+  - networking
+---
+## Contents
```

Comment:
I would suggest to remove the header "Contents". Just start the introductory paragraph directly with "This technical blog post gives..."

Reply:
Done